types/minio/index.d.ts

```diff
@@ -368,7 +368,7 @@ export class Client {
     setBucketPolicy(bucketName: string, bucketPolicy: string, callback: NoResultCallback): void;
     setBucketPolicy(bucketName: string, bucketPolicy: string): Promise<void>;
 
-    listenBucketNotification(bucketName: string, prefix: string, suffix: string, events: NotificationEvent[]): EventEmitter;
+    listenBucketNotification(bucketName: string, prefix: string, suffix: string, events: NotificationEvent[]): NotificationPoller;
```

Comment:
could you add a call to a `NotificationPoller` function [around here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/f46779ee53c8cac359072176c03bae06030916ee/types/minio/minio-tests.ts#L309) to satisfy the test check?

Reply:
Of course! Added [here]https://github.com/mateusm09/DefinitelyTyped/blob/720f5caca330c2ac04a8e260c79c9be1eef470ef/types/minio/minio-tests.ts#L310)